recipes/clone_prep_clear_config.rb

```diff
@@ -0,0 +1,17 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: clone_prep_clear_config
+#
+# Make a universal forwarder part of a host image.
+#
+# See http://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Makeauniversalforwarderpartofahostimage
+
+execute 'clone-prep-clear-config' do
+  command "#{node['splunk']['cmd']} clone-prep-clear-config"
+  user node['splunk']['user']
+  group node['splunk']['group']
+  only_if { ::File.exist? "#{node['splunk']['cmd']}" }
```

Comment:
I don't know that we want to guard this execution. If someone has configuration that for whatever reason the splunk cmd does not exist by this point, we would want chef to blow up.